patches/lxcfs-0001-hook.patch

```diff
@@ -10,6 +10,12 @@
 +    rm -Rf "${LXC_ROOTFS_MOUNT}/var/snap/lxd/common/var/lib/lxcfs"
 +    mkdir -p "${LXC_ROOTFS_MOUNT}/var/snap/lxd/common/var/lib/lxcfs"
 +    mount -n --bind /var/snap/lxd/common/var/lib/lxcfs "${LXC_ROOTFS_MOUNT}/var/snap/lxd/common/var/lib/lxcfs/"
++fi
++
++# For backward compatibility, hand `/var/lib/lxcfs` through the host to
++# the container being as the lxcfs mount point.
++if [ -d ${LXC_ROOTFS_MOUNT}/var/lib/lxcfs/ ]; then
++    mount -n --bind /var/snap/lxd/common/var/lib/lxcfs ${LXC_ROOTFS_MOUNT}/var/lib/lxcfs/
```

Comment:
Can you make the path quoting match what we have for the hunk above?
Something like:
```
if [ -d "${LXC_ROOTFS_MOUNT}/var/lib/lxcfs/" ]; then
    mount -n --bind /var/snap/lxd/common/var/lib/lxcfs "${LXC_ROOTFS_MOUNT}/var/lib/lxcfs/"
fi
```

Reply:
@adglkh

Reply:
Thanks. fixed.